st2api/st2api/controllers/exp/aliasexecution.py

```diff
@@ -101,16 +110,17 @@ def _get_notify_field(self, payload):
         notify.on_complete = on_complete
         return notify
 
-    def _schedule_execution(self, action_alias_db, params, notify):
+    def _schedule_execution(self, action_alias_db, params, notify, context):
         try:
             # prior to shipping off the params cast them to the right type.
             params = action_param_utils.cast_params(action_ref=action_alias_db.action_ref,
                                                     params=params,
                                                     cast_overrides=CAST_OVERRIDES)
-            context = {
-                'action_alias_ref': reference.get_ref_from_model(action_alias_db),
```

Comment:
It would probably be better to always include those two attributes here.

Otherwise we need to add a docstring to the function which says that the `context` which is passed to this function must contain those two attributes.